include/riakc.hrl

```diff
@@ -68,6 +68,9 @@
                       head | deletedvclock |
                       {n_val, pos_integer()} |
                       {sloppy_quorum, boolean()}.
+-type preflist_item() :: {parition, non_neg_integer()} | {node, binary()} |
+                         {primary, boolean()}.
+-type preflist() :: [preflist_item()].
```

Comment:
Oh, I see in the test. Still, maybe each item should come back as a tuple, not a list? The number of items is known in each entry.